src/Bicep.Core/Semantics/ArmTemplateSemanticModel.cs

```diff
@@ -52,6 +52,7 @@ public ArmTemplateSemanticModel(ArmTemplateFile sourceFile)
 
                 return schemaUri.AbsolutePath switch
                 {
+                    // TODO: I think this is for decompiling, so add DSC schemas and map to DSC scope
```

Comment:
This is used for being able to reference compiled JSON files in a module statement - e.g. rather than:
```bicep
module foo 'mod.bicep' = {
```

You can also do:
```bicep
module foo 'mod.json' = {
```

Reply:
Ahh, thank you. I wondered.

Reply:
NP, I wasn't sure how relevant this would be DSC. It would block being able to pull DSC modules from a registry, but that may not be your focus right now!